codec_specs.md

```diff
@@ -1,21 +1,59 @@
 ---
 layout: default
 ---
-# CodecID
+# Codec Specifications
 
-As an additional resource to this page Haali has created a [list of codec IDs in a PDF](http://haali.su/mkv/codecs.pdf).
+Each TrackEntry inside [Matroska]({{site.baseurl}}/index.html) MUST reference a defined [Codec ID]({{site.baseurl}}/index.html#CodecID). This Codec ID is a registered identifier that represents the codec used to encode the data stored within the Track. The codec works with the coded data in the stream, but MAY also require some form of codec initialisation. Matroska uses two different Elements to store information for codec initialisation:
 
-For each TrackEntry inside [matroska]({{site.baseurl}}/index.html), there has to be a [CodecID]({{site.baseurl}}/index.html#CodecID) defined. This ID is represent the codec used to encode data in the Track. The codec works with the coded data in the stream, but also with some codec initialisation. There are 2 different kind of codec "initialisation":
+- CodecPrivate in the TrackEntry
+- CodecState in the BlockGroup
 
-*   CodecPrivate in the TrackEntry
-*   CodecState in the BlockGroup
-
-Each of these elements contain the same kind of data. And these data depend on the codec used.
+Each of these Elements contain the same kind of data. And these data depend on the codec used.
 
 Important Note:
 
-The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (matroska-devel at lists.matroska.org).
+The intention behind this list is `NOT` to list all existing audio and video codecs, but rather to list those codecs that are `currently supported` in Matroska and therefore need a well defined codec ID so that all developers supporting Matroska will use the same ID. If you feel we missed support for a very important codec, please tell us on our development mailing list (cellar at ietf.org).
```

Comment:
I agree with @retokromer. Let's correct it now: lower-case "not" without backticks. Additionally I'd remove the backticks around "currently supported". It's supposed to be an emphasis, I guess, but we usually use backticks for official terminology like element names.